README.md

```diff
@@ -243,9 +243,36 @@ This render prop wraps its return value in a list item (`<li>`) and a `<button>`
 | ----------- | ------- | ------------------------------------------------------------------------------------------------------------------------- |
 | active      | Boolean | Whether the current pagination item being iterated over corresponds to the active gallery item.                           |
 | activeIndex | Number  | The index of the currently active gallery item.                                                                           |
-| i           | Number  | The index of the current pagination item being iterated over.                                                             |
+| index       | Number  | The index of the current pagination item being iterated over.                                  |
 | item        | Any     | The current pagination item being iterated over, as defined by the Array fed to the `<Gallery>` component's `items` prop. |
 
+**Example usage of `children` render props**
+```jsx
+<GalleryPagination
+  renderPaginationItem={({ index, active }) => (
+    <GalleryPaginationItem index={index} active={active} key={index}>
+      <span>{index + 1}</span>
+    </GalleryPaginationItem>
+  )}
+/>
+```
+### &lt;GalleryPaginationItem&gt;
+
+Used as children for `<GalleryPagination>`. This component with a return value in a list item (`<li>`) and a `<button>`, and receives a handful of arguments:
+
+#### Props:
+
+
+| Prop               | Type     | Description                                                                                                     |
+| ------------------ | -------- | --------------------------------------------------------------------------------------------------------------- |
+| active             | Boolean  | Boolean to set the `<button>`'s `aria-current` attribute.                                                       |
+| buttonClassName    | String   | Set the `<button>` element's class.                                                                             |
+| buttonProps        | Object   | Pass props to the `<button>` element.                                                                           |
+| children           | JSX      | Pass children to the component to render them as children of the implicit `<button>` element.                     |
+| className          | String   | Set the `<li>` element's class.                                                                                  |
+| index              | Number   | This needs to be a unique key for the `<li>` element. Used to set the gallery's active item to the associated pagination item button clicked/ |
```

Comment:
Minor tweak here but it feels like an important one to me. Should we change the word "key" when describing the `index` prop, so that it cannot be confused with the `key` prop? Just thinking defensively here

Reply:
Changed this to `identifier`